packages/react-menu/src/components/MenuItem/MenuItem.types.ts

```diff
@@ -45,6 +51,12 @@ export interface MenuItemState extends MenuItemProps {
    */
   icon?: ObjectShorthandProps<React.HTMLAttributes<HTMLSpanElement>>;
 
+  /**
+   * A helper slot for alignment when a menu item is used with selectable menuitems
+   * Avoid using this slot as a replacement for MenuItemCheckbox and MenuItemRadio components
+   */
+  checkmark: ShorthandProps<React.HTMLAttributes<HTMLElement>>;
```

Comment:
Shouldn't this be `ObjectShorthandProps<React.HTMLAttributes<HTMLSpanElement>>`?

Reply:
I have no idea man.... `as` kills typings :)

Reply:
```suggestion
  checkmark: ObjectShorthandProps<React.HTMLAttributes<HTMLElement>>;
```

?